plotly/figure_factory/_facet_grid.py

```diff
@@ -0,0 +1,1073 @@
+                                **kwargs_marker
+                            ),
+                            **kwargs
+                        )
+                    else:
+                        trace = graph_objs.Scatter(
+                            x=group[x].tolist(),
+                            y=group[y].tolist(),
+                            mode='markers',
+                            type=trace_type,
+                            name=color_val,
+                            marker=dict(
+                                color=colormap[color_val],
+                                size=size,
+                                **kwargs_marker
+                            ),
+                            showlegend=False,
+                            **kwargs
+                        )
+                    fig.append_trace(trace, row_count + 1, col_count + 1)
+                if row_count == 0:
+                    label = _return_label(col_values[col_count],
+                                          facet_col_labels, facet_col)
+                    annotations.append(
+                        _annotation_dict(label, col_count + 1, num_of_cols,
+                                         row_col='col', flipped=flipped_cols)
+                        )
+            label = _return_label(row_values[row_count],
+                                  facet_row_labels, facet_row)
+            annotations.append(
+                _annotation_dict(label, num_of_rows - row_count, num_of_rows,
+                                 row_col='row', flipped=flipped_rows)
+                )
+
+    # add annotations
+    fig['layout']['annotations'] = annotations
+
+    return fig
+
+
+def _facet_grid_color_numerical(df, x, y, facet_row, facet_col, color_name,
+                                colormap, height, width, num_of_rows,
+                                num_of_cols, facet_row_labels,
+                                facet_col_labels, trace_type, size,
+                                flipped_rows, flipped_cols, show_boxes,
+                                kwargs, kwargs_marker):
+
+    fig = make_subplots(rows=num_of_rows, cols=num_of_cols,
+                        shared_xaxes=True, shared_yaxes=True,
+                        horizontal_spacing=SUBPLOT_SPACING,
+                        vertical_spacing=SUBPLOT_SPACING, print_grid=False)
+
+    annotations = []
+    if not facet_row and not facet_col:
+        trace = graph_objs.Scatter(
+            x=df[x].tolist(),
+            y=df[y].tolist(),
+            mode='markers',
+            type=trace_type,
+            marker=dict(
+                color=df[color_name],
+                size=size,
+                colorscale=colormap,
+                showscale=True,
+                **kwargs_marker
+            ),
+            **kwargs
+        )
+        fig.append_trace(trace, 1, 1)
+
+    elif facet_row and not facet_col:
+        groups_by_facet_row = list(df.groupby(facet_row))
```

Comment:
We could even allow users to pass in their own "`cut`" arrays, so that they could make custom groups (for example, they might want to groups like `[[0, 1], [1, 10], [10, 100], [100, 1000]]`